### DISCUSSION TITLE
how to send attachment from my form?

### DISCUSSION THREAD

**Discussion by OlgaVolos** [emailjs-com/help#93] [Q&A]
I have a paid account. The form does not work for sending a file. Does anyone have a working code? How to process the file and form data to send the file? For React

**Comment by xr0master** [comment]
Since there is no explains what was done, this is a vague question.

To send the file, you need to add an input file to the form with the name attribute. Let's say your variable is "my_variable".

```html
<input type="file" name="my_variable" />
```

In the template (in the EmailJS dashboard), add the Dynamic File Form Attachment with the "my_variable" Parameter Name. Save template.

How to submit the form, on the link below
https://www.emailjs.com/docs/examples/reactjs/

**Reply by PayamTaherirostami** [reply]
@xr0master  I have the same question and I was wondering if you could help me. I have an array that I want to pass as a "csv" file into my attachment. Here is what I have:
 let myArray = [1,2,3]
   emailjs.send('serviceID', 'templateID',{Files:myArray},'public key')
when I get the email, the attachment looks like Chinese or Japanese! any Idea?
